src/Features/Core/Portable/Completion/CommonCompletionItem.cs

```diff
@@ -12,41 +12,6 @@ namespace Microsoft.CodeAnalysis.Completion
 {
     internal static class CommonCompletionItem
     {
```

Comment:
I assume this is safe to do because of the ExternalAcesss for F#.

Reply:
Yes.